mix.lock

```diff
@@ -0,0 +1,9 @@
+%{
```

Comment:
what do you mean @bterone , this file is the lock version for the `nimble_phx_gen_template` itself 🤔

The nimble_phx_gen_template required the Phoenix dependencyy

![image](https://user-images.githubusercontent.com/11751745/94983594-7e64aa80-056e-11eb-8012-bd8f09aef88d.png)

Reply:
Oh, I meant just the `.lock` file, not the `mix.exs`, since we're copying it from Github it also adds the lock file to the `deps` folder which I didn't see in other dependency folders. I thought it builds everything on the project you're generating on so it'd be in that `mix.lock` file. Nevermind, it doesn't seem to put it when you publish a package 😂